core/src/main/resources/db/cgds.sql

```diff
@@ -724,8 +727,34 @@ CREATE TABLE `clinical_event_data` (
   FOREIGN KEY (`CLINICAL_EVENT_ID`) REFERENCES `clinical_event` (`CLINICAL_EVENT_ID`) ON DELETE CASCADE
 );
 
+CREATE TABLE `gene_panel` (
+    `INTERNAL_ID` int(11) NOT NULL auto_increment,
+    `STABLE_ID` varchar(255) NOT NULL,
+    `DESCRIPTION` mediumtext,
+    PRIMARY KEY (`INTERNAL_ID`),
+    UNIQUE (`STABLE_ID`)
+);
+
+CREATE TABLE `gene_panel_list` (
+    `INTERNAL_ID` int(11) NOT NULL,
+    `GENE_ID` int(255) NOT NULL,
+    PRIMARY KEY (`INTERNAL_ID`, `GENE_ID`),
+    FOREIGN KEY (`INTERNAL_ID`) REFERENCES `gene_panel` (`INTERNAL_ID`) ON DELETE CASCADE,
+    FOREIGN KEY (`GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`) ON DELETE CASCADE
+);
+
+CREATE TABLE `gene_panel_profile_map` (
+    `SAMPLE_ID` int(11) NOT NULL,
+    `PROFILE_ID` int(11) NOT NULL,
+    `PANEL_ID` int(11) NOT NULL,
+    PRIMARY KEY (`SAMPLE_ID`, `PROFILE_ID`),
```

Comment:
I am trying to understand the issue just raised by Pieter. Is this the scenario? : a sample is processed as part of a batch of samples using a particular (limited) gene panel; then the results come back as invalid for the sample or showing no mutation events at all ; the sample_profile table does not show the sample even though it was processed with a gene panel so we are unable to register the gene_panel ; finally in the oncoprint we cannot distinguish for such a sample whether a gene was "tested but not mutated" versus "untested" ?
Secondary question: can a sample be sent for sequencing but then be excluded from the generated results because of detection of problems during processing? If this does happen, perhaps we should not consider the gene panel genes as being "tested" because the test itself was invalidated.